core/templates/pages/exploration-editor-page/services/autosave-info-modals.service.spec.ts

```diff
@@ -16,11 +16,13 @@
  * @fileoverview Unit tests for autosaveInfoModalsService.
  */
 
-import { LocalStorageService } from 'services/local-storage.service';
 import { fakeAsync, flushMicrotasks, TestBed } from '@angular/core/testing';
+import { HttpClientModule } from '@angular/common/http';
 import { NgbModal, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+
```

Comment:
Unnecessary line.

Reply:
I'm adding a space between Angular imports and our imports, this is not enforced rn, but I think it is better and it mimics how we handle imports in Python.